src/components/Calendar/CheckinModal.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import { Modal, Button, Switch } from 'antd'
+import { CheckCircleOutlined, CheckCircleTwoTone } from '@ant-design/icons';
+import '../../less/calendar.less'
+
+const CheckinModal = (props) => {
+
+    if(!props.details.start) {
+        return(
+            <>
+                loading
+            </>
+        )
+    }
+
+    console.log(props.details)
```

Comment:
Hey, I was wondering if these two functions can become one since they both do the same thing. Maybe call it handleModalVisibility or something? And then link it to both 'ok' and 'cancel' buttons?